docs/user_guides/debugging/features.rst

```diff
@@ -1,11 +1,12 @@
 Supported GDB Features
 ======================
 
-Currently, the following debugging features are available:
+Currently the following debugging features are available:
 
 - Source location (filename and line number).
 - Setting break points by the line number.
 - Stepping over break points.
+- Providing information about local variables.
```

Comment:
This list should be removed. It was task for @vlad-perevezentsev in https://github.com/IntelPython/numba-dppy/issues/453 with fix from https://github.com/IntelPython/numba-dppy/pull/459.

Reply:
Do we need to remove all the list?